src/components/energiatodistus-haku/querybuilder/query-inputs/unformatted-number-input.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
```

Comment:
Muuten mielestäni siisti ratkaisu, mutta vähän häiritsee että tämä nimiavaruus on kahta riviä luukuottamatta sama kuin number-input.svelte.

Yritin tehdä siitä geneerisemmän laittamalla formatin exportin taakse ja kuljettamalla propsit tästä sinne $$props -taialla, mutta siitä ei tullut oikein siisti.

Reply:
Juu, tää on sinällään hankala kun näitä komponentteja ei oikein pysty partially applyttämään. Yksi vaihtoehto olisi tehdä erillinen format-agnostinen number-input.svelte ja siihen sitten erikoistaa päälle tuollaisilla yksinkertaisilla formattednumber/unformattednumber komponenteilla, jotka syöttävät tuolle number-inputille nuo olemassaolevat exportit ja halutun formattifunkkarin.

Reply:
Nyt on geneerinen number-input.svelte, ja siitä extendattu unformatted-number-input.svelte. Erillistä formatted-number-inputtia ei tarvitse tehdä, kun number-inputilla on järkevät defaultit.